app/static/styles/main.css

```diff
@@ -31,6 +31,22 @@ body {
   padding-top: calc(var(--padding) * 2);
 }
 
+.navbar-section li a {
+  font-weight: 500;
+  font-size: 16px;
+  text-decoration: none;
+
+  color: var(--lightgrey);
+
+}
+
+.navbar-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px 4%;
+}
+
```

Comment:
All this is related to the previous comment on implementing a feature altogether.

Reply:
Sorry, I'm not sure what you mean by this comment.

Reply:
Oh actually nvm, these are rules for the navbar; my bad.